test/dune

```diff
@@ -0,0 +1,4 @@
+(library
+ (name Test)
+ (modes melange)
+ (libraries bs_react_intl_ppx))
```

Comment:
This should be `melange-react-intl-ppx` right? or `MelangeReactIntlPpx` (I am not sure how this was working before)

Reply:
https://github.com/ahrefs/bs-react-intl-ppx/blob/1cea617f9b80a4f417c05ef5657515ba9cc3b60f/test/dune#L4